core/src/test/java/com/klaytn/caver/common/transaction/EthereumAccessListTest.java

```diff
@@ -1279,4 +1280,282 @@ public void throwException_NotDefined_ChainID() {
         }
     }
 
+    public static class signWithKeyTest {
+        @Rule
+        public ExpectedException expectedException = ExpectedException.none();
+
+        static Caver caver = new Caver();
+
+        static AbstractKeyring coupledKeyring;
+        static AbstractKeyring deCoupledKeyring;
+        static String privateKey = "0x45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8";
+
+        static String gas = "0x1e241";
+        static String gasPrice = "0x0a";
+        static String to = "0x095e7baea6a6c7c4c2dfeb977efac326af552d87";
+        static String chainID = "0x1";
+        static String input = "0x616263646566";
+        static String value = "0x1";
+        static String nonce = "0x4";
+
+        static AccessList accessList = new AccessList(Arrays.asList(
+                new AccessTuple("0x0000000000000000000000000000000000000001",
+                        Arrays.asList(
+                                "0x0000000000000000000000000000000000000000000000000000000000000000"
+                        )),
+                new AccessTuple("0x284e47e6130523b2507ba38cea17dd40a20a0cd0",
+                        Arrays.asList(
+                                "0xd2b691e13d4c3754fbe5dc75439f25dd11a908d89f5bbc55cd5bc4978f078b7c",
+                                "0xd2db659067b2b322f7010149472b81f172b9e331e1831ebee11c7b73facb0761"
+                        )),
+                new AccessTuple("0x0000000000000000000000000000000000000003",
+                        Arrays.asList(
+                                "0x46d62a62fb985e2e7691a9044b8fae9149311c7f3dcf669265fe5c96072ba4fc",
+                                "0x6eab5ba2ea17e1ef4eac404d25f1fe9224421e3b639aec73d3b99c39f0983681"
+                        ))
+        ));
+
+        public static EthereumAccessList createEthereumAccessList() {
+            return caver.transaction.ethereumAccessList.create(
+                    TxPropertyBuilder.ethereumAccessList()
+                            .setTo(to)
+                            .setNonce(nonce)
+                            .setGas(gas)
+                            .setGasPrice(gasPrice)
+                            .setInput(input)
+                            .setChainId(chainID)
+                            .setValue(value)
+                            .setAccessList(accessList)
+            );
+        }
+
+        static SignatureData expectedSignature = new SignatureData(
+                "0x01",
```

Comment:
Can you add a test case to check handling for "0x00" case?

Reply:
Hi jamie. 
I added test case for it and there was an issue about it. Thanks really.
Please take a look [this commit](https://github.com/klaytn/caver-java/pull/291/commits/05205f6401ba3ad6f464c64c36713245370aabb5).

I used the same [test cases from caver-js](https://github.com/klaytn/caver-js/blob/03d469b25acb9de5574336b621c80d2645f75d81/test/packages/caver.transaction/ethereumAccessList.js#L238-L273)